frontend-ui/src/components/Login.js

```diff
@@ -0,0 +1,126 @@
+import React, {useEffect, useState} from "react";
+import axios from 'axios';
+import {useHistory} from "react-router-dom";
+
+function LoginNavbar() {
+    return (
+        <div className={"gray-navbar"}>
+            <p className={"left-item"}>LOGIN </p>
+        </div>
+    );
+}
+const Login = () => {
+    let history = useHistory();
+    const [credentials, setCredentials] = useState({
+        email: '',
+        password: '',
+        rememberMe: ''
+    })
+    useEffect(() => {
+        if (localStorage.checkbox && localStorage.checkbox !== "") {
+            setCredentials({
+                ...credentials,
+                ...{
+                    email: localStorage.email,
+                    rememberMe: localStorage.checkbox
+                }
+            })
+        } else {
+            setCredentials({
+                ...credentials,
+                ...{
+                    email: "",
+                    rememberMe: ""
+                }
+            })
+        }
+    }, []);
+    const lsRememberMe = () => {
+        if (credentials.rememberMe && credentials.email !== "") {
+            localStorage.email = credentials.email;
+            localStorage.checkbox = credentials.rememberMe;
+        } else {
+            localStorage.email = "";
+            localStorage.checkbox = "";
+        }
+    }
+    const handleEmailChange = (e) => {
+        setCredentials({
+            ...credentials,
+            ...{email: e.target.value.toString()}
+        })
+    }
+    const handleRememberMeChange = (e) => {
+        setCredentials({
+            ...credentials,
+            ...{rememberMe: e.target.checked}
+        })
+    }
+    const handlePasswordChange = (e) => {
+        setCredentials({
+            ...credentials,
+            ...{password: e.target.value.toString()}
+        })
+    }
+    const handleLoginClick = async (e) => {
+        lsRememberMe();
+        const url = "http://localhost:8080/api/login"
```

Comment:
Try to not use hardcoded URL's. Maybe use something like this. `const url = HOST_URL + '/api/login'`
HOST_URL variable should be common for all api methods. You can hardcoded value for this attribute for now, but be aware once you deploy this app on server, value for HOST_URL should be provided from environment variables since it depends on environment. Also, store this URL in shared files, maybe in api/auth.js or something like this (they should not be declared in component files)